### PR TITLE
fix: upgrade gen version now pulls image correctly

### PR DESCRIPTION
**Pull request by armandobelardo**
The generator upgrade command now pulls the latest image before running `getImage` I'd assumed this was handled within `getImage` as it was working locally

### CODE REVIEW

File: packages/cli/configuration/src/generators-yml/addGenerator.ts [hunk 1]
```diff
@@ -52,13 +52,29 @@ export async function upgradeGenerator({
     context: TaskContext;
 }): Promise<GeneratorsConfigurationSchema> {
     const normalizedGeneratorName = getGeneratorNameOrThrow(generatorName, context);
-    return upgradeGeneratorVersion({
+    context.logger.info(`${normalizedGeneratorName} ${groupName}`);
```
Comment (dsinghvi) [inline packages/cli/configuration/src/generators-yml/addGenerator.ts:55]: is this intentional?

Comment (armandobelardo) [reply packages/cli/configuration/src/generators-yml/addGenerator.ts:55]: nope, removed

File: packages/cli/configuration/src/generators-yml/addGenerator.ts [hunk 2]
```diff
@@ -22,6 +21,7 @@ async function getLatestGeneratorVersion(generatorName: string, context: TaskCon
     const docker = new Docker();
     let image;
     try {
+        await docker.pull(`${generatorName}`);
```
[the remark this answers is not in the document]
Comment (armandobelardo) [reply packages/cli/configuration/src/generators-yml/addGenerator.ts:24]: just add and generate upgrade cli commands

[the remark this answers is not in the document]
Comment (armandobelardo) [reply packages/cli/configuration/src/generators-yml/addGenerator.ts:24]: and when it fails it just returns null and goes with whatever the cli path did originally (e.g. for add it falls back to the invocations file)

**Pull request merged**